src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1294,13 +1294,19 @@ public function get_event_count( $type = null ) {
 			return 0;
 		}
 
+		$activity_type = 'event';
+
+		if ( ! empty( $this->meta['content_type'] ) ) {
+			$activity_type = $this->meta['content_type'];
```

Comment:
Should we have a allowed content type array?

Reply:
I'm not sure we need one at this point in the code.